src/renderer/components/watch-video-info/watch-video-info.js

```diff
@@ -268,15 +256,32 @@ export default defineComponent({
         playlistIndex: this.getPlaylistIndex(),
         playlistReverse: this.getPlaylistReverse(),
         playlistShuffle: this.getPlaylistShuffle(),
-        playlistLoop: this.getPlaylistLoop()
-      })
-    },
-
-    toggleSave: function () {
-      if (this.inFavoritesPlaylist) {
-        this.removeFromPlaylist()
-      } else {
-        this.addToPlaylist()
+        playlistLoop: this.getPlaylistLoop(),
+      }
+      // Only play video in non playlist mode when user playlist detected
+      if (this.inUserPlaylist) {
+        Object.assign(payload, {
+          playlistId: null,
+          playlistIndex: null,
+          playlistReverse: null,
+          playlistShuffle: null,
+          playlistLoop: null,
+        })
+      }
+      this.openInExternalPlayer(payload)
+    },
+
+    handleFormatChange: function (format) {
+      switch (format) {
+        case 'dash':
+          this.$parent.enableDashFormat()
+          break
+        case 'legacy':
+          this.$parent.enableLegacyFormat()
+          break
+        case 'audio':
+          this.$parent.enableAudioFormat()
+          break
```

Comment:
This function was removed a while ago (the template uses `$emit`, instead of calling those functions with `$parent`), I'm guessing this was because development was merged into here, instead of this branch being rebased on top of development.